.settings/org.scala-ide.sdt.core.prefs

```diff
@@ -0,0 +1,6 @@
+#Generated by sbteclipse
+#Mon Jul 04 19:04:02 CEST 2016
+scala.compiler.additionalParams=-encoding utf8
+deprecation=true
+unchecked=true
+scala.compiler.useProjectSettings=true
```

Comment:
We should .gitignore `org.scala-ide.sdt.core.prefs`, which is generated by sbteclipse.